City-of-Helsinki/drupal-helfi-platform-config
# UHF-5936 Hel.fi test content

# [UHF-5936](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5936)

## What was done
* Created a base structure for the test content with Default content module
* Applied patch for the Default content to handle manual imports

## How to install and test
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-5936_helfi_test_content --with-all-dependencies`
* Run `make drush-cr`
* Run `make shell`
  * In the container, run: `drush en -y helfi_test_content`
  * Once the module is installed, you can see explore the test content in `/en/dc-helfi-platform-test-content`. For example: https://helfi-kymp.docker.so/en/urban-environment-and-traffic/dc-helfi-platform-test-content
